creator-node/src/index.ts

```diff
@@ -165,6 +166,9 @@ const startAppForPrimary = async () => {
       }
     }
   })
+
+  // do not await this, this should just run in background for now
+  DiskManager.sweepSubdirectoriesInFiles()
```

Comment:
idk how i feel about how this is called but it's simple and it works

Reply:
placement makes sense to me